examples/qasm.ipynb

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
<p>I'm a bit confused by this sentence</p><blockquote>&nbsp;QASM skips the include command for the file qelib1.inc and converts all custom gates specified in them directly into QuTiP gates.</blockquote><p>If we skip <code>qelib1.inc</code>, how can we convert the gates specified in this header file?</p><p> </p><blockquote>It is recommended to use the <code>read_qasm</code> function in this mode.  </blockquote><p>Could you elaborate a bit more here? Do we support other mode? Or is there API functions other than <code>read_qasm</code> that we would like the users to know?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
>Could you elaborate a bit more here? Do we support other mode? Or is there API functions 
  other than read_qasm that we would like the users to know?

I am omitting this comment for now since the usage of alternative modes is not super useful. I will still keep the `mode` argument as it's potentially useful in the futue